kythe/go/platform/kzip/kzip.go

```diff
@@ -105,6 +106,7 @@ const (
 	PROTO Encoding = 2
```

Comment:
I believe these are generally named:

`<Prefix>Enum`, e.g.
```
EncProto
EncJson
EncAll
```

Reply:
Full names and it should be "JSON"

```
const (
  EncodingAll = iota
  EncodingJSON
  Encoding Proto
)
```